api/swagger.yml

```
@@ -923,6 +923,14 @@ components:
             with an external auth service.
           type: string
           enum: [none, simplified, external]
+        username_placeholder:
+          description: |
+            Placeholder text to display in the username field of the login form.
+          type: string
+        password_placeholder:
```

nit: (none-blocker!)

```suggestion
        password_placeholder_label:
```

---

changed to `password_ui_placeholder`